c/tests/test_minimal_cpp.cpp

```diff
@@ -168,7 +168,9 @@ test_edge_sorting()
     assert(tables.edges.num_rows == (tsk_size_t) n - 1);
 
     /* Make sure the edges are unsorted */
-    ret = tsk_table_collection_check_integrity(&tables, TSK_CHECK_EDGE_ORDERING);
+    /* Not calling TSK_CHECK_TREES so casting is safe */
+    ret = (tsk_id_t) tsk_table_collection_check_integrity(
```

Comment:
We don't need to cast here, ret is declared as a tsk_id_t and the return value is already tsk_id_t. I think we should change the type of ``ret`` to int here (I probably changed this, before I realised it was a mistake to change ``ret`` values to tsk_id_t

Reply:
Good catch, I've made `ret` an `int` here.